client/packages/reports/src/utils.ts

```diff
@@ -1,10 +0,0 @@
-import { LocaleKey, useTranslation } from '@common/intl';
-
-export const translateReportName = (
-  t: ReturnType<typeof useTranslation>,
-  reportName: string
-) => {
-  const key =
-    `report.${reportName.replace(/ /g, '-').toLowerCase()}` as LocaleKey;
-  return key == t(key) ? reportName : t(key);
-};
```

Comment:
Don't need this function anymore.

Also moved functionality into IntlUtils as report name translations are used in both system/ and report/